apps/nextjs-website/src/editorialComponents/Newsroom/Newsroom.tsx

```diff
@@ -99,7 +99,9 @@ const Item = (props: INewsroomItem) => {
           {new Intl.DateTimeFormat(locale, options).format(date)}
         </Typography>
       )}
-      <Typography variant='h6'>{title}</Typography>
+      <Typography variant='h6' sx={{ minHeight: 55 }}>
```

Comment:
This solution is fragile: if the title is on 3 rows, the alignment gets lost
![image](https://github.com/pagopa/developer-portal/assets/39835990/e8944a11-0141-4065-9cb8-ea21e428d645).
I think that you should add a flex container (flex-direction: column) around the title and the link, with content justified "space-between"